src/extensions/Directory.Build.props.old

```diff
@@ -108,12 +108,12 @@
   </PropertyGroup>-->
 
   <ItemGroup Condition="'$(IsWinUI)' != 'true' and '$(IsUWP)' != 'true' and '$(UnoCompatibility)' != 'UWP'">
-    <PackageReference Include="Uno.WinUI" Version="4.0.9" >
+    <PackageReference Include="Uno.WinUI" Version="4.0.11" >
```

Comment:
Remove this file entirely ?

Reply:
done - removed the old app template and example app